forem/forem
# Don't skip indexing for old articles

## What type of PR is this? (check all applicable)


## Description
Remove `published_at.to_i < 1_500_000_000 ||` from articles `skip_indexing` for indexing old articles 
Because of this condition old articles skip indexing
example: view-source:https://dev.to/bhaskar_vk/how-to-use-twitters-search-rest-api-most-effectively


## Related Tickets & Documents



- Related Issue #19039 
- Closes #19039 

## QA Instructions, Screenshots, Recordings

this condition skips indexing for all articles published before July 14, 2017
to check, you can open any articles before that date and check that these tags are not there
```
<meta name="robots" content="noindex">
<meta name="robots" content="nofollow">
```

example: view-source:https://dev.to/bhaskar_vk/how-to-use-twitters-search-rest-api-most-effectively




### UI accessibility concerns?

_If your PR includes UI changes, please replace this line with details on how
accessibility is impacted and tested. For more info, check out the
[Forem Accessibility Docs](https://developers.forem.com/frontend/accessibility)._

## Added/updated tests?

      have not been included_

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

      [Storybook](https://storybook.forem.com/) (for Crayons components)
      updated. I have filled out the
      [Changes Requested](https://github.com/forem/admin-docs/issues/new?assignees=&labels=&template=changes_requested.md)
      issue template so Community Success can help update the Admin Docs
      appropriately.
      [`CHANGELOG.md`](https://github.com/forem/forem/tree/main/CHANGELOG.md)
      or in a [forem.dev](http://forem.dev) post
      replace this line with details on why this change doesn't need to be
      shared_

## [optional] Are there any post deployment tasks we need to perform?

## [optional] What gif best describes this PR or how it makes you feel?

![alt_text](https://media.giphy.com/media/l4KibWpBGWchSqCRy/giphy.gif)
